tests/cypress/support/commands.js

```diff
@@ -49,6 +49,30 @@ Cypress.Commands.add("testPageAccessibility", () => {
   }
 });
 
+Cypress.Commands.add("ensureAvailableReport", () => {
```

Comment:
NIT: When I read the name of the command, I assumed it would create a work plan if none existed. This is doing almost the opposite. Maybe a name more like `archiveAnyExistingWorkPlans`?

Reply:
good call, my good sir. It shall be renamed!